.gitmodules

```
@@ -7,3 +7,6 @@
 [submodule "mmtf_spec"]
 	path = mmtf_spec
 	url = https://github.com/rcsb/mmtf
+[submodule "pybind11"]
+	path = pybind11
+	url = git@github.com:pybind/pybind11.git
```

```suggestion
	url = https://github.com/pybind/pybind11
```
I think CI doesn't like the ssh access. Maybe better to use https as for the others?